packages/db/convex/discord.ts

```diff
@@ -375,3 +377,15 @@ export const resolveThread = internalMutation({
     });
   },
 });
+
+export const getThreadByDiscordThreadId = internalQuery({
+  args: {
+    discordThreadId: v.string(),
+  },
+  handler: async ({ db }, { discordThreadId }) => {
+    return await db
+      .query("threads")
+      .withIndex("id", (q) => q.eq("id", discordThreadId))
+      .unique();
+  },
+});
```

Comment:
I'm guessing this is only used by places where we can look up the discord thread in the mutation about to kick off an action (e.g. from somewhere in discord.js)